tests/integration/docusaurus/tutorials/getting-started/getting_started.py

```diff
@@ -14,7 +14,7 @@
 
 # First configure a new Datasource and add to DataContext
 datasource_yaml = f"""
-name: data__dir
+name: getting_started_datasource
 class_name: Datasource
 module_name: great_expectations.datasource
 execution_engine:
```

Comment:
Note : The names of the DataConnectors were kept as is because they are generated automatically by the CLI and not configured by the user. Is this still ok for our purposes @anthonyburdi ?